app/sdil/controllers/RegistrationController.scala

```diff
@@ -41,23 +37,24 @@ import sdil.uniform.SaveForLaterPersistence
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import uk.gov.hmrc.uniform.FormHtml
+import uk.gov.hmrc.uniform.playutil._
+import uk.gov.hmrc.uniform.webmonad._
 import views.html.uniform
 
 import scala.concurrent.{ExecutionContext, Future}
 
 
 class RegistrationController(
-                              val messagesApi: MessagesApi,
                               authorisedAction: AuthorisedAction,
                               sdilConnector: SoftDrinksIndustryLevyConnector,
                               registeredAction: RegisteredAction,
-                              cache: RegistrationFormDataCache
-                            )(
-                              implicit
-                              val config: AppConfig,
-                              val ec: ExecutionContext
-                            )
-  extends SdilWMController with FrontendController {
+                              cache: RegistrationFormDataCache,
+                              mcc: MessagesControllerComponents)
+                            (implicit val config: AppConfig, val ec: ExecutionContext)
+  extends FrontendController(mcc) with SdilWMController with I18nSupport {
+
+  override implicit lazy val messages = MessagesImpl(mcc.langs.availables.head, messagesApi)
```

Comment:
In future we need to find a way to get rid of this as .head is probably not the best practice but we know "en" will always be there

Reply:
yes so I am to create a ticket aimed at this. to remove the need for creating this MessagesImpl and somehow ideally deal with this implicitly.

Reply:
https://jira.tools.tax.service.gov.uk/browse/DLS-562